### ISSUE TITLE
Anaconda-mode causes error 60

### ISSUE THREAD

**Issue by 3rd3** [syl20bnr/spacemacs#6937]
#### Description
`anaconda-mode` throws some error

#### Reproduction guide
- Just edit a python file.
*Observed behaviour:*
```
*anaconda-response*
# status: (:error (error connection-failed failed with code 60
 :host 127.0.0.1 :service 9000))
# point: 1

```

*Expected behaviour:*
DESCRIPTION OF THE EXPECTED BEHAVIOUR

#### System Info
- OS: darwin
- Emacs: 25.1.50.1
- Spacemacs: 0.105.22
- Spacemacs branch: master (rev. 9f9faa4)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(emacs-lisp syntax-checking auto-completion semantic
            (latex :variables latex-enable-auto-fill t)
            org git github version-control extra-langs markdown emacs-lisp haskell html javascript shell-scripts java
            (haskell :variables haskell-enable-ghc-mod-support nil haskell-enable-ghci-ng-support t haskell-enable-hindent-style "gibiansky" haskell-enable-shm-support nil)
            (c-c   :variables c-c  -enable-clang-support t)
            (python :variables python-test-runner 'pytest)
            (clojure :variables clojure-enable-fancify-symbols t)
            (shell :variables shell-default-height 20 shell-default-position 'bottom shell-enable-smart-eshell t shell-default-shell 'eshell)
            evil-snipe)
```

**Comment by TheBB** [comment]
Well, I can't reproduce this on 24.5. Have you tried with a stable release Emacs?

**Comment by TheBB** [comment]
Consider also reporting upstream. It's unlikely to be a Spacemacs problem.

**Comment by 3rd3** [comment]
Thanks for the hint. The bug does not seem to occur on 24.5. I will report that elsewhere. Unfortunately, 24.5 has some issues that are resolved in 25.1, but I guess being able to edit Python code is more important for now. :)
